package.json

```diff
@@ -1,35 +1,32 @@
 {
-  "name": "timers",
-  "type": "module",
-  "version": "1.1.0",
-  "scripts": {
-    "dev": "astro dev",
-    "start": "astro dev",
-    "build": "astro build",
-    "preview": "astro preview",
-    "astro": "astro"
-  },
-  "dependencies": {
-    "@astrojs/react": "2.3.2",
-    "@astrojs/tailwind": "4.0.0",
-    "@types/react": "18.2.20",
-    "@types/react-dom": "18.2.7",
-    "astro": "^2.10.12",
-    "react": "18.2.0",
-    "react-canvas-confetti": "1.4.0",
-    "react-dom": "18.2.0",
-    "solid-js": "^1.7.6",
-    "tailwindcss": "3.3.3"
-  },
-  "devDependencies": {
-    "@technologiestiftung/semantic-release-config": "1.2.3",
-    "cz-conventional-changelog": "3.3.0",
-    "netlify-cli": "16.1.0",
-    "semantic-release": "21.0.9"
-  },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
-  }
+	"name": "timers",
+	"type": "module",
+	"version": "1.1.0",
+	"scripts": {
+		"dev": "astro dev",
+		"start": "astro dev",
+		"build": "astro build",
+		"preview": "astro preview",
+		"astro": "astro"
+	},
+	"dependencies": {
+		"@astrojs/react": "2.3.2",
+		"@astrojs/tailwind": "4.0.0",
+		"@types/react": "18.2.20",
+		"@types/react-dom": "18.2.7",
```

Comment:
Still have react deps in dependencies and devdependencies. In case you want to fully get rid of em. :)

Reply:
Damn.